honeycomb-core/src/cmap/dim2/basic_ops.rs

```diff
@@ -21,7 +21,7 @@ use crate::{
 };
 
 use std::collections::BTreeSet;
-
+use std::sync::atomic::{AtomicBool, Ordering};
```

Comment:
Why do not use transactions here too?

Reply:
Ok, there's two things:

- for unused dart storage, `AtomicBool` is enough because there's no situation / operation where the validity of a dart is a critical information. This could change if we implement algorithms that delete part of the mesh sporadically, but even then it doesn't matter: What is important is that all beta function of the deleted dart are set to 0, not that it is marked. marks matter when fetching all cells after (or before) an algorithm is executed.

- for beta values: these should probably be transactional too. the biggest consequence would be that some method (e.g. attribute collection methods) need to take a transaction as argument as the `atomically` section provided byt he stm crate should not be nested